runtime/flang/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ SET(CMAKE_ASM_FLAGS "${CFLAGS} ${ASM_OPTIONS}" )
 if(${SANITIZER})
   set(CMAKE_SHARED_LINKER_FLAGS " -Wl,--enable-new-dtags -fuse-ld=lld  -fsanitize=address -shared-libasan -g -Wl,--build-id=sha1 -no-flang-libs ")
 else()
-  set(CMAKE_SHARED_LINKER_FLAGS " -no-flang-libs ")
+  set(CMAKE_SHARED_LINKER_FLAGS " -no-flang-libs -Wl,--enable-new-dtags,--rpath,$ORIGIN:$ORIGIN/../../lib")
```

Comment:
This should be guarded by AOMP_STANDALONE_BUILD=0. But I suggest doing so in build_flang_runtime.sh as there is a guard present there already. Maybe you can add a variable here that will be empty when AOMP_STANDALONE_BUILD=1.

line 36: 
```set(CMAKE_SHARED_LINKER_FLAGS " -no-flang-libs ${$OPENMP_EXTRAS_LINKER_FLAGS}")```

build_flang_runtime.sh:
```
if [ "$AOMP_STANDALONE_BUILD" == 0 ]; then
  MYCMAKEOPTS="$MYCMAKEOPTS $OPENMP_EXTRAS_ORIGIN_RPATH
  -DENABLE_DEVEL_PACKAGE=ON -DENABLE_RUN_PACKAGE=ON
+ -DOPENMP_EXTRAS_LINKER_FLAGS=-Wl,--enable-new-dtags,--rpath,$ORIGIN:$ORIGIN/../../lib"
fi
```

Reply:
Updated the patch as per the comments
Following patch will set the variable and pass as build argument 
https://github.com/ROCm-Developer-Tools/aomp/pull/637